sunset/code/modules/client/client_defines.dm

```diff
@@ -0,0 +1,2 @@
+/client
+	control_freak = CONTROL_FREAK_ALL | CONTROL_FREAK_MACROS
```

Comment:
?

Reply:
Both of these are from the original PR, but I removed control_freak_all as it seemed to do nothing.